tests/runtests.py

```diff
@@ -15,6 +15,7 @@
 from django.conf import settings
 settings.configure(
     DATABASES={'default': {'ENGINE': 'django.db.backends.sqlite3'}},
+    DEFAULT_AUTO_FIELD='django.db.models.BigAutoField',
```

Comment:
Why is this different from apps.py ?

Reply:
And why is it needed at all ?

Reply:
> Why is this different from apps.py ?

Using `BigAutoField` in `DjangoCommentsAdminConfig` would generate migrations. In tests we can use `BigAutoField` without any side-effects. See [release notes](https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys) for more details.